src/getCoinCombination.js

```diff
@@ -7,15 +7,15 @@
  */
 function getCoinCombination(cents) {
   let currentAmount = cents;
-  const values = [1, 5, 10, 25];
+  const values = [25, 10, 5, 1]; // Починаємо з кварталів
   const coins = [0, 0, 0, 0];
 
-  for (let i = 3; i >= 0; i--) {
+  for (let i = 0; i < values.length; i++) {
     coins[i] = Math.floor(currentAmount / values[i]);
     currentAmount -= coins[i] * values[i];
   }
 
-  return coins;
+  return [coins[3], coins[2], coins[1], coins[0]];
```

Comment:
The return statement reverses the order of the coins array. This means the function will return the number of pennies, nickels, dimes, and quarters, instead of quarters, dimes, nickels, and pennies. If the task requires returning the coins in the order of quarters, dimes, nickels, and pennies, you should return the 'coins' array as is.